vit-servicing-station-tests/Cargo.toml

```diff
@@ -15,7 +15,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.53"
 structopt = "0.3.14"
 thiserror = "1.0"
-tokio = { version = "0.2.21", features = ["macros"] }
+tokio = { version = "0.3.3", features = ["macros", "sync", "fs"] }
```

Comment:
This is still at 0.3?

Reply:
Missed this one. And one hyper there too. It is updated now.